diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 6b0e6c249..c5e64cac5 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -53,7 +53,6 @@ add_subdirectory(csync)
 add_subdirectory(libsync)
 if (NOT BUILD_LIBRARIES_ONLY)
     add_subdirectory(cmd)
-    add_subdirectory(gui)
 
     if (WITH_CRASHREPORTER)
         add_subdirectory(3rdparty/libcrashreporter-qt)
@@ -68,9 +67,6 @@ if(KRAZY2_EXECUTABLE)
                        ${PROJECT_SOURCE_DIR}/src/libsync/*.ui
                        ${PROJECT_SOURCE_DIR}/src/libsync/*.h
                        ${PROJECT_SOURCE_DIR}/src/libsync/*.cpp
-                       ${PROJECT_SOURCE_DIR}/src/gui/*.ui
-                       ${PROJECT_SOURCE_DIR}/src/gui/*.h
-                       ${PROJECT_SOURCE_DIR}/src/gui/*.cpp
                        ${PROJECT_SOURCE_DIR}/src/cmd/*.h
                        ${PROJECT_SOURCE_DIR}/src/cmd/*.cpp
 )
