diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index a4d06844a..fdcf1a981 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -30,35 +30,7 @@ set_package_properties(Qt5Concurrent PROPERTIES
     TYPE REQUIRED
 )
 
-find_package(Qt5WebEngineWidgets ${REQUIRED_QT_VERSION} CONFIG QUIET)
-if(APPLE)
-    set_package_properties(Qt5WebEngineWidgets PROPERTIES
-        DESCRIPTION "Qt5 WebEngineWidgets component."
-        TYPE RECOMMENDED
-    )
-else()
-    set_package_properties(Qt5WebEngineWidgets PROPERTIES
-        DESCRIPTION "Qt5 WebEngineWidgets component."
-        TYPE REQUIRED
-    )
-endif()
-
-find_package(Qt5WebEngine ${REQUIRED_QT_VERSION} CONFIG QUIET)
-if(APPLE)
-    set_package_properties(Qt5WebEngine PROPERTIES
-        DESCRIPTION "Qt5 WebEngine component."
-        TYPE RECOMMENDED
-    )
-else()
-    set_package_properties(Qt5WebEngine PROPERTIES
-        DESCRIPTION "Qt5 WebEngine component."
-        TYPE REQUIRED
-    )
-endif()
-
-if(Qt5WebEngine_FOUND AND Qt5WebEngineWidgets_FOUND)
-  add_compile_definitions(WITH_WEBENGINE=1)
-endif()
+add_compile_definitions(WITH_WEBENGINE=0)
 
 get_target_property (QT_QMAKE_EXECUTABLE Qt5::qmake IMPORTED_LOCATION)
 message(STATUS "Using Qt ${Qt5Core_VERSION} (${QT_QMAKE_EXECUTABLE})")
